app/assets/javascripts/discourse/app/lib/plugin-api.js

```diff
@@ -1716,9 +1722,10 @@ class PluginApi {
    * @param {string=} arg.href - The href attribute for the link.
    * @param {string} arg.title - The title attribute for the link.
    * @param {string} arg.text - The text to display for the link.
+   * @param {Boolean} [secondary] - Determines whether the section link should be added to the main or secondary section in the "More..." links drawer.
    */
-  addCommunitySectionLink(arg) {
-    addSectionLink(arg);
+  addCommunitySectionLink(arg, secondary) {
+    addSectionLinkToCommunity(arg, secondary);
```

Comment:
I think a better alias for `addSectionLinkToCommunity` would be `addCustomCommunitySectionLink` so you don't have it too similar to `addCommunitySectionLink`

Reply:
Agreed here. I've updated the alias.